modules/contact/include/ReferenceResidualProblem.h

```diff
@@ -58,6 +58,13 @@ class ReferenceResidualProblem : public FEProblem
   std::vector<std::string> _refResidVarNames;
   ///@}
 
+  ///@{
+  /// List of grouped solution variable names whose reference residuals will be stored,
+  /// and the residual variable names that will store them.
+  std::vector<std::string> _group_solnVarNames;
```

Comment:
Follow the MOOSE naming conventions for member variables (something like `_group_soln_var_names`),

Reply:
It looks like we weren't following those conventions on some of the other variables before (like _solnVarNames). It probably makes sense to go ahead and change those as well.